style.css

```diff
@@ -0,0 +1,74 @@
+.toolbar {
```

Comment:
- [ ] Please I highly suggest adding responsive breakpoints per 
      project requirements. I am sorry it was not mentioned earlier. 😄 
       The minimum mobile screen size should be `375px`
       The minimum desktop screen size should be `768px`

- [ ] Please I highly suggest adding responsive values to your CSS 
       rules per project requirements. I am sorry it was not 
       mentioned earlier. 😄 
       Example: `width: 90%`.

This is the general and project requirements [link](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md)